lib/helpers/answerHelper.js

```diff
@@ -10,9 +10,7 @@ function dateFormatter(date) {
 }
 
 function isValidDate(str) {
-    const d = Date.parse(str);
-    // eslint-disable-next-line no-restricted-globals
-    return !isNaN(d);
```

Comment:
use `Number.isNaN(d)` and it won't complain!

Reply:
> use `!Number.isNaN(d)` and it won't complain!

It wasn't a good way to do it in any case. It's been removed in my [other commit](b01f910)